test/facets/embalmer-facet.spec.ts

```diff
@@ -40,44 +38,105 @@ describe("Contract: EmbalmerFacet", () => {
         );
       });
 
-      it("should emit InitializeSarcophagus()", async () => {
-        const { embalmerFacet, initializeTx } = await createSarcoFixture(
-          { shares, threshold, skipFinalize: true },
+      it("should emit createSarcophagus()", async () => {
```

Comment:
Minor nitpick but I think conventionally events are PascalCased

Reply:
good catch, updating this. What's interesting is that the event name was PascalCased in the contract, but this spec still passed -- the `expect.to.emit` must not check for capitalization.